daml/splice-amulet/daml/Splice/AmuletConfig.daml

```diff
@@ -70,6 +105,12 @@ data PackageConfig = PackageConfig
     walletPayments : Text
   deriving (Show, Eq)
 
+-- | Package configs are not constrained at the Daml level to maximize flexibility.
+-- In particular, empty packages names might be used in the future to indicate that
```

Comment:
```suggestion
-- In particular, empty packages versions might be used in the future to indicate that
```
you can't set a name. it's a record with the field values being versions. Which does mean you could validate that it's an empty version or a valid one. I have no strong opinion on whether that's worth doing.